release-1.8/release_notes_draft.md

```diff
@@ -69,58 +69,55 @@ integrations with external Key Management Systems.
 [SIG Cluster Lifecycle][] is responsible for the user experience of deploying,
 upgrading, and deleting clusters.
 
-For the 1.8 release SIG Cluster Lifecycle continued to focus on expanding the
+For the 1.8 release, SIG Cluster Lifecycle continued to focus on expanding the
 capabilities of kubeadm, which is both a user-facing tool to manage clusters
 and a building block for higher-level provisioning systems. Starting
-with the 1.8 release kubeadm supports a new upgrade command and has alpha
+with the 1.8 release, kubeadm supports a new upgrade command and includes alpha
 support for self hosting the cluster control plane.
 
 [SIG Cluster Lifecycle]: https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle
 
-
 ### SIG Node
 
-[SIG Node][] is responsible for the components which support the controlled
-interactions between pods and host resources as well as managing the lifecycle
+[SIG Node][] is responsible for the components that support the controlled
+interactions between pods and host resources, and manage the lifecycle
 of pods scheduled on a node.
 
-For the 1.8 release SIG Node continued to focus
-on supporting the broadest set of workload types, including hardware and performance
-sensitive workloads such as data analytics and deep learning, while delivering
-incremental improvements to node reliability.
+For the 1.8 release, SIG Node continued to focus
+on supporting a broad set of workload types, including hardware and performance
+sensitive workloads such as data analytics and deep learning. The SIG also
+delivered incremental improvements to node reliability.
```

Comment:
For the 1.8 release, SIG Node continued to focus on a broad set of workload types, including hardware and performance sensitive workloads, such as data analytics and deep learning. 

(tried to cut down the number of words in the sentence)

Reply:
Nice! Thank you!

Reply:
Didn't take the comma, though, because then the appositive would also modify "hardware"